rpki-validator/src/main/java/net/ripe/rpki/validator3/domain/validation/ValidatedRpkiObjects.java

```diff
@@ -292,6 +299,28 @@ public void add(TrustAnchorData trustAnchorData, Key key, CertificateRepositoryO
                         pkInfo
                 ));
             }
+
+            DateTime expiresAt = null;
+            if (object instanceof RpkiSignedObject) {
+                RpkiSignedObject obj = (RpkiSignedObject) object;
+                expiresAt = obj.getNotValidAfter();
```

Comment:
Do we need to check both expiration time of the EE certificate and CMS-specific 'notValidaAfter' here? According to ROS report they are different on out manifests, so it would make sense to double-check here.

Reply:
Fixed